Lib/fontTools/varLib/instancer.py

```diff
@@ -0,0 +1,1031 @@
+""" Partially instantiate a variable font.
+
+This is similar to fontTools.varLib.mutator, but instead of creating full
```

Comment:
Not sure it's helpful to define this in terms of mutator, just define what it is directly.